.changeset/tricky-actors-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@blobscan/web": patch
+---
+
+The Tooltip component now includes extra logic to prevent it from overflowing.
```

Comment:
Let's follow the standard and only use past tense verbs please

```suggestion
Added additional logic to the `Tooltip` component to prevent overflow.
```